docs/resources/repository_network_access_policy.md

```diff
@@ -4,8 +4,7 @@ page_title: "cyral_repository_network_access_policy Resource - terraform-provide
 subcategory: ""
 description: |-
     Manages the network access policy of a repository. Network access policies are also known as the Network Shield https://cyral.com/docs/manage-repositories/network-shield/. This feature is supported for the following repository types:
-      - sqlserver
-      - oracle
+    sqlserveroracle
```

Comment:
We probably want to revert this change?

Reply:
This was introduced since the migration to the new version of the TF docs. I tried to fix it, but it seems like a bug in the TF docs itself. As this is just a description header that is not used to render, I'm ok to keep it as it is generated automatically.